### PR TITLE
[Docs] Post process stub files

### PR DESCRIPTION
**Pull request by sunethwarna**
**📝 Description**
Post processes created stub files to have proper Kratos module imports.

**🆕 Changelog**
Please summarize the changes in one list to generate the changelog:
- Add post processing to stub files

### CODE REVIEW

File: scripts/post_install/stub_generation.py [hunk 1]
```diff
@@ -169,6 +168,18 @@ def __append_cpp_module_stub(cpp_module_file_path: Path, python_module_file_path
 
     return cpp_module_name
 
+def PostProcessGeneratedStubFiles():
+    kratos_installation_path = (Path(sys.argv[1])).absolute()
+    files = glob.glob(str(kratos_installation_path) + '/**/*.pyi', recursive=True)
```
Comment (matekelemen) [inline scripts/post_install/stub_generation.py]: `pathlib` can do [recursive globbing](https://docs.python.org/3/library/pathlib.html#pathlib.Path.rglob):
```py
kratos_installation_path.rglob("*.pyi")
```

File: scripts/post_install/stub_generation.py [hunk 2]
```diff
@@ -169,6 +168,18 @@ def __append_cpp_module_stub(cpp_module_file_path: Path, python_module_file_path
 
     return cpp_module_name
 
+def PostProcessGeneratedStubFiles():
+    kratos_installation_path = (Path(sys.argv[1])).absolute()
```
Comment (matekelemen) [inline scripts/post_install/stub_generation.py]:
```suggestion
    kratos_installation_path = Path(sys.argv[1]).absolute()

```

File: scripts/post_install/stub_generation.py (hunk 1 above)
Comment (sunethwarna) [reply scripts/post_install/stub_generation.py]: done :)

**Review by matekelemen — APPROVED** [review]
👌

**Pull request merged**